Add new topics (conditionally change a band's visibility and reorder table cells) (#7)
VeraGorbyliova/dotnet-eud: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-expression-bindings/conditionally-change-a-bands-visibility-expression-bindings.md

---
title: Conditionally Change a Band's Visibility
owner: Vera Gorbyliova
seealso: []
---
# Conditionally Change a Band's Visibility

> This topic describes how to change report band visibility.

Reviewer: ```suggestion
This topic describes how to change the report band's visibility.
```

Set a band's **Visible** property to an expression to conditionally change the band's visibility based on a field's value or a parameter.

The report created in this tutorial contains two Detail **sub-bands** with different report controls. These sub-bands are used to display discontinued and current products.   

![1](../../../../../images/conditionally-change-a-bands-visibility.png)

The steps below demonstrate how to change a band's visibility based on a field's value.

> 1. Select the required band and switch to the **Properties** panel. Choose **Expressions** ![](../../../../../images/conditionally-change-a-bands-visibility-expression-button.png) and click the **Visible** property's ellipsis button.

Reviewer: ```suggestion
1. Select a band and switch to the **Properties** panel. Choose **Expressions** ![](../../../../../images/conditionally-change-a-bands-visibility-expression-button.png) and click the **Visible** property's ellipsis button.
```

	![](../../../../../images/conditionally-change-a-bands-visibility-visible-button.png)

> 2. In the invoked **Expression Editor**, specify the required expression.

Reviewer: ```suggestion
2. In the invoked **Expression Editor**, specify the expression.
```
